std/std.zig

```diff
@@ -7,7 +7,7 @@ pub const Buffer = @import("buffer.zig").Buffer;
 pub const BufferOutStream = @import("io.zig").BufferOutStream;
 pub const DynLib = @import("dynamic_library.zig").DynLib;
 pub const HashMap = @import("hash_map.zig").HashMap;
-pub const LinkedList = @import("linked_list.zig").LinkedList;
+pub const LinkedList = @import("linked_list.zig").TailQueue;
```

Comment:
These names should match, one way or the other. I'd prefer `LinkedList` to `TailQueue`. `LinkedList` and `ArrayList` go well together.

Reply:
The issue ( #2384 ) is that there are many types of LinkedList, and people should make an informed choice about which type they want.
I kept `std.LinkedList` as the old type to avoid making this a **breaking** PR.